<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Reinhold Kainhofer &lt;reinhold@kainhofer.com&gt;</author>
 <comment>Configuration page for printing incidences.</comment>
 <class>CalPrintIncidenceConfig_Base</class>
 <widget class="QWidget" name="CalPrintIncidenceConfig_Base">
  <layout class="QVBoxLayout">
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Print incidence options:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mIncludeInfoGroup">
     <property name="title">
      <string>Include Information</string>
     </property>
     <property name="selectedId" stdset="0">
      <number>-1</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="mShowDetails">
        <property name="toolTip">
         <string>Print incidence details</string>
        </property>
        <property name="whatsThis">
         <string>Check this box if you want to print more details for the incidence, including the visibility and secrecy properties.</string>
        </property>
        <property name="text">
         <string>Detai&amp;ls (visiblility, secrecy, etc.)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mShowAttendees">
        <property name="toolTip">
         <string>Print the attendees</string>
        </property>
        <property name="whatsThis">
         <string>Check this box if you want the attendees list included on the print-out.</string>
        </property>
        <property name="text">
         <string>&amp;Attendees</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mShowSubitemsNotes">
        <property name="toolTip">
         <string>Print the notes</string>
        </property>
        <property name="whatsThis">
         <string>Check this box if you want the incidence notes included on the print-out.</string>
        </property>
        <property name="text">
         <string>&amp;Notes, subitems</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mShowAttachments">
        <property name="toolTip">
         <string>Print the attachments</string>
        </property>
        <property name="whatsThis">
         <string>Check this box if you want the attachment information included on the print-out. The attachments themselves are not printed.</string>
        </property>
        <property name="text">
         <string>Attach&amp;ments</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mGeneralGroup">
     <property name="title">
      <string comment="@title general print settings">General</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="mShowNoteLines">
        <property name="toolTip">
         <string>Check this option to draw note lines in empty areas.</string>
        </property>
        <property name="whatsThis">
         <string>Check this box if you want to draw note lines in the empty area of the print. This is useful if you want to hand-write notes on this print-out.</string>
        </property>
        <property name="text">
         <string>Show note &amp;lines</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mColors">
        <property name="toolTip">
         <string>Print in color</string>
        </property>
        <property name="whatsThis">
         <string>If you want to use colors to distinguish certain tags in the printed output, check this option.</string>
        </property>
        <property name="text">
         <string>&amp;Use colors</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mPrintFooter">
        <property name="toolTip">
         <string>Print a datetime footer on each page</string>
        </property>
        <property name="whatsThis">
         <string>Check this box if you want to print a small footer on each page that contains the date of the print.</string>
        </property>
        <property name="text">
         <string>Print &amp;Footer</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
